app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -700,7 +700,7 @@ class NimbusExperimentSerializer(
         required=False,
         validators=[
             NimbusStatusTransitionValidator(
-                transitions=NimbusConstants.VALID_STATUS_TRANSITIONS,
+                transitions=NimbusConstants.VALID_STATUS_NEXT_VALUES,
```

Comment:
Wait, why would we use the `status_next` transitions on the `status` field?

Reply:
The status transitions should be identical to the status next transitions

Reply:
I guess the question is, why were we using VALID_STATUS_TRANSITIONS only for draft <-> preview?

Reply:
The assumption that I made is that the `NimbusStatusTransitionValidator` should test _all_ status transitions, not just that one

Reply:
> The status transitions should be identical to the status next transitions

I don't think that's true?

Reply:
> The assumption that I made is that the `NimbusStatusTransitionValidator` should test _all_ status transitions, not just that one

Ah yeah see, two different concerns.  The V5 API acts as a broker between the UI and the database, so it only needs to be concerned with what the UI can do, and the UI only allows users to move between Draft and Preview. 

If we wanted to have a broader status transition validator, that would live at the model layer so that anywhere in the code that tries to change the status is forced to go through it so statuses are always valid.  That is a pattern we've explored in the past, but the big problem with that is that every so often you want to actually break out of the usual flow and hop between states that aren't usually moved between, maybe in tests, maybe in code, or most importantly, from the admin where you need to be able to reset some state.  Actually that just happened with me this week.  So in general when we've tried to do like global state transition enforcement it hurt more than it helped.  

The other way we've kind of soft approached this is with the way we're storing the states as those Q object constants and then just referring to them everywhere.  That way you can easily look at the list of all states as those constants and compare them to the diagrams more easily.

So yeah I get what you're thinking of, but it's not something I'd recommend, and even if it was, it wouldn't go in here.